web/src/components/PaginatedList/PaginatedList.tsx

```diff
@@ -13,35 +12,22 @@ type TParams<P> = P & {
 
 interface IProps<T, P> {
   dataCy?: string;
-  itemComponent({item}: {item: T}): ReactElement;
   params: TParams<P>;
   query: UseQuery<any>;
+  itemComponent({item}: {item: T}): ReactElement;
 }
 
 interface IId {
   id: string;
 }
 
 const PaginatedList = <T extends IId, P>({dataCy, itemComponent: ItemComponent, params, query}: IProps<T, P>) => {
-  const {hasNext, hasPrev, isEmpty, isFetching, isLoading, list, loadNext, loadPrev} = usePagination<T, typeof params>(
-    query,
-    params
-  );
+  const pagination = usePagination<T, typeof params>(query, params);
 
   return (
-    <Pagination
-      emptyComponent={<Empty />}
-      hasNext={hasNext}
-      hasPrev={hasPrev}
-      isEmpty={isEmpty}
-      isFetching={isFetching}
-      isLoading={isLoading}
-      loadingComponent={<Loading />}
-      loadNext={loadNext}
-      loadPrev={loadPrev}
-    >
+    <Pagination emptyComponent={<Empty />} loadingComponent={<Loading />} {...pagination}>
```

Comment:
I see you added a `generic` for the `Pagination` component. but you are not passing it here.